docs/cel2/l2-operator-guide.md

```diff
@@ -1,437 +1,30 @@
 ---
-title: Celo L2 Migration Guide
-description: How to migrate Celo nodes from L1 to L2
+title: Celo L2 node operator guide
```

Comment:
```suggestion
title: Celo L2 Node Operator Guide
```

Reply:
We don't have a consistent title style, but I've seen capital letters more often.